src/Http/Middleware/FlarumPromotionHeader.php

```diff
@@ -21,7 +21,7 @@ class FlarumPromotionHeader implements Middleware
 
     public function __construct(Config $config)
     {
-        $this->enabled = $config['poweredByHeader'] ?? true;
+        $this->enabled = $config['headers']['poweredByHeader'] ?? true;
```

Comment:
What if there is no headers key? Should we use `Arr::get`?

Reply:
That's a good point, I'll fix that here and the security header PR